components/tag/index.css

```diff
@@ -12,195 +12,190 @@
 	--spectrum-tag-font-size: var(--spectrum-font-size-100);
 	--spectrum-tag-icon-size: var(--spectrum-workflow-icon-size-100);
+	--spectrum-tag-corner-radius: var(--spectrum-corner-radius-medium-size-medium);
 
-	--spectrum-tag-clear-button-spacing-inline-start: var(--spectrum-text-to-visual-100);
+	--spectrum-tag-clear-button-size: var(--spectrum-cross-icon-size-100);
 	--spectrum-tag-clear-button-spacing-block: var(--spectrum-tag-top-to-cross-icon-medium);
+	--spectrum-tag-label-to-clear-icon: var(--spectrum-tag-label-to-clear-icon-medium);
+	--spectrum-tag-edge-to-clear-icon: var(--spectrum-tag-edge-to-clear-icon-medium);
+
+	--spectrum-tag-visual-spacing-inline-start: var(--spectrum-component-edge-to-visual-100);
+	--spectrum-tag-visual-spacing-inline-end: var(--spectrum-text-to-visual-100);
 
 	--spectrum-tag-icon-spacing-block-start: var(--spectrum-component-top-to-workflow-icon-100);
 	--spectrum-tag-icon-spacing-block-end: var(--spectrum-component-top-to-workflow-icon-100);
-	--spectrum-tag-icon-spacing-inline-end: var(--spectrum-text-to-visual-100);
 
-	--spectrum-tag-avatar-spacing-block-start: var(--spectrum-tag-top-to-avatar-medium); /* 6px 9px */
+	--spectrum-tag-avatar-size: var(--spectrum-avatar-size-100);
+	--spectrum-tag-avatar-spacing-block-start: var(--spectrum-tag-top-to-avatar-medium);
 	--spectrum-tag-avatar-spacing-block-end: var(--spectrum-tag-top-to-avatar-medium);
-	--spectrum-tag-avatar-spacing-inline-end: var(--spectrum-text-to-visual-100);
 
-	--spectrum-tag-label-spacing-block: var(--spectrum-component-top-to-text-100);
+	--spectrum-tag-thumbnail-size: var(--spectrum-thumbnail-size-75);
 
-	/* tokens based on theme and t-shirt size */
-	--spectrum-tag-corner-radius: var(--spectrum-tag-size-medium-corner-radius);
-	--spectrum-tag-spacing-inline-start: var(--spectrum-tag-size-medium-spacing-inline-start);
-	--spectrum-tag-label-spacing-inline-end: var(--spectrum-tag-size-medium-label-spacing-inline-end);
-	--spectrum-tag-clear-button-spacing-inline-end: var(--spectrum-tag-size-medium-clear-button-spacing-inline-end);
+	--spectrum-tag-label-spacing-block: var(--spectrum-component-top-to-text-100);
+	--spectrum-tag-label-spacing-inline: var(--spectrum-component-edge-to-text-100);
+
+	/* @passthrough start for nested components */
+	--mod-avatar-inline-size: var(--mod-tag-avatar-size, var(--spectrum-tag-avatar-size));
+	--mod-avatar-block-size: var(--mod-tag-avatar-size, var(--spectrum-tag-avatar-size));
+	--mod-thumbnail-size: var(--mod-tag-thumbnail-size, var(--spectrum-tag-thumbnail-size));
+	--mod-clear-button-width: var(--mod-tag-clear-button-size, var(--spectrum-tag-clear-button-size));
+	--mod-clear-button-height: var(--mod-tag-clear-button-size, var(--spectrum-tag-clear-button-size));
+	--mod-clear-button-background-color: transparent;
+	--mod-clear-button-icon-color: currentColor;
+	--mod-clear-button-icon-color-hover: currentColor;
+	--mod-clear-button-icon-color-key-focus: currentColor;
```

Comment:
Using a `--mod` passthrough to set the icon color instead of setting the color on `.spectrum-Tag-clearButton`, I also noticed in S1 that if the clear button is hovered or focused it was turning a black color which didn't look great for emphasized or selected states, it _should_ now just inherit the content color from the label.

S1:
![image](https://github.com/user-attachments/assets/88b84e08-d9a8-4f2d-ac39-50c27e716fa3)

S2:
<img width="109" alt="image" src="https://github.com/user-attachments/assets/9b3d8830-7c6c-4dd9-9910-81f14ec63fe2" />